content/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile.md

```diff
@@ -83,3 +83,20 @@ You can pin repositories that you want to feature, such as those that are freque
 1. Click **Save pins**.
 
 {% endif %}
+
+## Changing your organization's profile picture
+
```

Comment:
It'd be good to add some introductory text here and explain the options, similar to what's in "[Personalizing your profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#changing-your-profile-picture)."

```suggestion

When you create an organization, {% data variables.product.product_name %} provides you with a randomly generated "identicon." The identicon is generated from a hash of your organization's user ID, so there's no way to control its color or pattern.

You can replace the identicon with an image that represents your organization. To replace the image, you can upload a new image or use a Gravatar image.

```